nilearn/plotting/surf_plotting.py

```diff
@@ -446,7 +447,12 @@ def _plot_surf_matplotlib(coords, faces, surf_map=None, bg_map=None,
     limits = [coords.min(), coords.max()]
 
     # set view
-    elev, azim = _set_view_plot_surf_matplotlib(hemi, view)
+    if isinstance(view, str):
+        elev, azim = _set_view_plot_surf_matplotlib(hemi, view)
+    elif isinstance(view, collections.Sequence) and len(view) == 2:
+        elev, azim = view
+    else:
+        raise TypeError(f"view must be a string of a sequence of length 2. Got {view} instead")
```

Comment:
Thanks, @NicolasGensollen! I had one quick style question: do you have an opinion about whether I should change the documentation manually in surface plotting functions or whether I should attempt to change it through `@filldoc`? I think that the latter would be possible, but I might have to create a new entry the docdict in _utils/docs.py, e.g. `custom_view`. (Otherwise, some functions that use a view but that do not support a custom view might also have their docstrings erroneously changed.)

This may be a small concern; I'm just curious whether you would have a preference.